ca_gazebo/models/create_camera/model.sdf

```diff
@@ -0,0 +1,35 @@
+<?xml version='1.0'?>
+<sdf version='1.6'>
+  <model name='camera'>
+    <static>true</static>
+    <link name='link'>
+      <visual name='visual'>
+        <geometry>
+          <box>
+            <size>0.1 0.1 0.1</size>
+          </box>
+        </geometry>
+      </visual>
+      <sensor name='my_camera' type='camera'>
+        <camera>
+          <save enabled="true">
```

Comment:
Don't save the frame, it's not necessary.

Reply:
True. Addressed.